### PR TITLE
adapt for zsh commands

### PR DESCRIPTION
**Pull request by lianghongzhuo**
adapt for zsh commands

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -296,13 +296,11 @@ void displayButton(Button button, int button_id, std::string group, ImVec2 butto
         {
             if(button.keep_open)
             {
-                std::string extra = "; exec bash";
-                std::string cmd = button.path + extra;
-                execl("/usr/bin/gnome-terminal", "ControlpanelTerminal", "--" , "bash", "-c", cmd.c_str(), NULL);
+                system(("/usr/bin/gnome-terminal -- " + button.path).c_str());
```
Comment (v4hn) [inline src/main.cpp:299]: This doesn't keep the terminal open after the command finished, right?
That feature is the whole reason for this `if` branch.

I don't see a way around the `*sh -c "<cmd>; exec sh"` argument to `gnome-terminal` here.
If you need zsh for a specific script, why don't you specify it in the `scriptpanel.yaml` for the concrete item?

File: src/main.cpp [hunk 2]
```diff
@@ -296,13 +296,11 @@ void displayButton(Button button, int button_id, std::string group, ImVec2 butto
...
             }
             else
             {
-                execl("/bin/bash", "bash", "-c", button.path.c_str(), NULL);
+                system(button.path.c_str());
```
Comment (v4hn) [inline src/main.cpp:303]: In theory no complains about this change, but it would be nice to keep both branches consistent.

**Review by v4hn — CHANGES_REQUESTED** [review]

File: src/main.cpp (hunk 2 above)
Comment (lianghongzhuo) [reply src/main.cpp:303]: true, I tested on the master branch, also works for zsh scripts.
close this pr.

Comment (lianghongzhuo) [reply src/main.cpp:303]: thank you for the instant review :)

**Pull request closed without merging**